targets/unity-v2/source/Shared/Public/PluginManager.cs

```diff
@@ -103,5 +112,26 @@ private ITransportPlugin CreatePlayFabTransportPlugin()
 
             return transport;
         }
+#if NET_4_6   
+        private IOneDSTransportPlugin CreateOneDSTransportPlugin()
+        {
+            IOneDSTransportPlugin transport = null;
+#if !UNITY_WSA && !UNITY_WP8
+            if (PlayFabSettings.RequestType == WebRequestType.HttpWebRequest)
+                transport = new OneDsWebRequestPlugin();
+#endif
+#if UNITY_2017_2_OR_NEWER
+            if (PlayFabSettings.RequestType == WebRequestType.UnityWww)
+                transport = new OneDsWwwPlugin();
```

Comment:
Todd updated the standard IHTTP equivalent of this function, and simplified and improved the logic with the idea that we're shifting towards UnityWebRequest being the more stable default. (WWW doesn't even exist anymore in 2018.3)

Please check that out, and update this sequence to match.